Task1_UserAPI/MeetingManager.Tests/UserCrudTest.cs

```diff
@@ -0,0 +1,160 @@
+        }
+
+        [TestMethod]
+        public async Task CanInsertUser()
+        {
+            var userData = new UserModel() { Email = "Test5@mail.com", FirstName = "Test5", LastName = "Test5", Password = "123456789" };
+            var userModel = await userService.CreateAsync(userData);
+            Assert.AreEqual(5, users.Count);
+            Assert.AreEqual(userData.FirstName, users[4].FirstName);
+            Assert.AreNotEqual("123456789", users[4].Password);
+        }
+
+        [TestMethod]
+        public async Task CanDeleteUser()
+        {
+            await userService.DeleteAsync(3);
+            Assert.AreEqual(3, users.Count);
+        }
+
+        [TestMethod]
+        public async Task CanNotDeleteUserByNotExistingId()
+        {
+            await userService.DeleteAsync(7);
+            Assert.AreEqual(4, users.Count);
+        }
+
+        [TestMethod]
+        public async Task CanUpdateUser()
+        {
+            var userData = new UserModel() { Id = 3, Email = "UpdateTest3@mail.com", FirstName = "UpdateTest3", LastName = "UpdateTest3", Password = "Update123456789" };
+            var userModel = await userService.UpdateAsync(userData);
+            Assert.IsNotNull(userModel);
+            Assert.AreEqual(userData.Email, users[2].Email);
+        }
+
+        [TestMethod]
+        public async Task CanNotUpdateUserByNotExistingId()
+        {
+            var userData = new UserModel() { Id = 7, Email = "UpdateTest7@mail.com", FirstName = "UpdateTest7", LastName = "UpdateTest7", Password = "Update123456789" };
+            var userModel = await userService.UpdateAsync(userData);
+            Assert.IsNull(userModel);
+        }
+
+        private void SetupMock(Mock<IUserRepository> mockRepository)
+        {
+            mockRepository.Setup(mr => mr.GetAllAsync())
+                .Returns(Task.Run(() => users));
+
+            mockRepository.Setup(mr => mr.GetOneAsync(It.IsAny<int>()))
+                .Returns((int id) => Task.Run(() => users.FirstOrDefault(u => u.Id == id)));
+
+            mockRepository.Setup(mr => mr.CreateAsync(It.IsAny<User>()))
+                .Returns((User userData) => Task.Run(() =>
+                {
+                    userData.Id = users.Count;
+                    users.Add(userData);
+                    return userData;
+                }));
+
+            mockRepository.Setup(mr => mr.DeleteAsync(It.IsAny<int>()))
+                .Returns((int id) => Task.Run(() =>
+                {
+                    var user = users.FirstOrDefault(u => u.Id == id);
+                    if (user != null)
+                    {
+                        users.Remove(user);
+                    }
+                }));
+
+            mockRepository.Setup(mr => mr.UpdateAsync(It.IsAny<User>()))
+                .Returns((User userData) => Task.Run(() =>
+                {
+                    var user = users.FirstOrDefault(u => u.Id == userData.Id);
```

Comment:
you have the same condition above
please, add private GetUser(int userId) method